src/app/backend/apihandler.go

```diff
@@ -140,6 +145,24 @@ func (apiHandler *ApiHandler) handleGetReplicaSetDetail(
 	response.WriteHeaderAndEntity(http.StatusCreated, result)
 }
 
+// Handles update of Replica Set API call.
+func (apiHandler *ApiHandler) handleUpdateReplicaSet(
+	request *restful.Request, response *restful.Response) {
+
+	replicaSetDetail := new(ReplicaSetDetail)
```

Comment:
Yes they are in URL and I will use them when I change current solution to send only information about what should be updated.